src/window.rs

```diff
@@ -1396,6 +1396,19 @@ impl Window {
             .maybe_wait_on_main(|w| w.drag_resize_window(direction))
     }
 
+    /// Show window menu at current mouse position.
+    ///
+    /// There's no guarantee that calling this method show the window menu.
+    /// 
```

Comment:
```suggestion
```
As we discussed, this isn't the case for Windows, other platforms should add documentation for this in "Platform-specific"s when the implementation is added.